Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -163,7 +163,7 @@ class BrowserViewController: UIViewController {
     }
 
     fileprivate var shouldShowDefaultBrowserPromo: Bool { profile.prefs.intForKey(PrefsKeys.IntroSeen) == nil }
-    fileprivate var shouldShowWhatsNewPageScreen: Bool { whatsNewDataProvider.shouldShowWhatsNewPage }
+    fileprivate var shouldShowWhatsNewPageScreen: Bool { !User.shared.firstTime && whatsNewDataProvider.shouldShowWhatsNewPage }
```

Comment:
Wouldn't this still attempts to show the page for the user the second time it opens the app?

Wondering if instead we don't need some logic that, the first time the user opens the app stores the current version so that when it's checked it sees as no update happened 🤔